docs/ares_mkquery.3

```diff
@@ -14,7 +14,7 @@ int ares_mkquery(const char *\fIname\fP, int \fIdnsclass\fP, int \fItype\fP,
                  int *\fIbuflen\fP)
 .fi
 .SH DESCRIPTION
-Deprecated function. See \fIares_create_query(3)\fP instead!
+Deprecated function. See \fIares_dns_record_create(3)\fP instead!
```

Comment:
Maybe:

> This function is deprecated as of c-ares 1.22.0. Please see \fIares_dns_record_create(3)\fP.

Reply:
Were both functions deprecated at the same version? Because one had a deprecation note for the other, it seems strange that they were both deprecated at the same version.

Reply:
hmm, no. But the function to use got changed.  I'm actually not sure the version the original deprecation occurred.  I guess we can omit that since it wasn't previously documented.

Reply:
I will work on this PR in a few days, I'll try to dig up the appearance of `ares_create_query` from git.

Reply:
Yes, I can't find the original deprecation either. That seems to be lost in time now. Moving forward, it would be good to document deprecation...

`ares_create_query` was introduced in 90364defb038d5d6b97acf9df9ed152d2393a863 which was first included in release v1.10. Of course, that does not mean that `ares_mkquery` was deprecated at that release.

The commands I used were:
```sh
# Find the first commit introducing ares_create_query
git log -S ares_create_query --reverse | head -n1
# Find the first release including the commit
git tag --sort=creatordate --contains 90364defb038d5d6b97acf9df9ed152d2393a863 --merged main | head -n1
```

Reply:
What I'll do is find the release that first introduced the previous deprecation and say that it's been deprecated at least since then.